cli/src/pcluster/models/imagebuilder.py

```diff
@@ -119,45 +118,23 @@ def _register_validators(self):
 # ---------------------- Dev Settings ---------------------- #
 
 
-class ChefCookbook(Resource):
-    """Represent the chef cookbook configuration for the ImageBuilder."""
-
-    def __init__(self, url: str, json: str):
-        super().__init__()
-        self.url = Param(url)
-        self.json = Param(json)
-        # TODO: add validator
-
-    def _register_validators(self):
-        self._add_validator(UrlValidator, url=self.url)
-
-
-class DevSettings(Resource):
+class DevSettings(BaseDevSettings):
     """Represent the dev settings configuration for the ImageBuilder."""
 
     def __init__(
         self,
         update_os_and_reboot: bool = None,
         disable_pcluster_component: bool = None,
-        chef_cookbook: ChefCookbook = None,
-        node_url: str = None,
-        aws_batch_cli_url: str = None,
         distribution_configuration_arn: str = None,
         terminate_instance_on_failure: bool = None,
+        **kwargs
```

Comment:
let's call them explicitly. in this way it's clear which parameters can be passed to the constructor of this class.

Reply:
I'd kept it unchanged for now. Because the caller of these initialization methods is only Marshmallow, instead of any manual calls. More over, it is easier to add another field to the BaseDevSettings with **kwargs